Auction Reporting.md

```diff
@@ -0,0 +1,110 @@
+# Auction Reporting
+## Overview
+In Microsoft Edge, we are supporting a limited version of ads reporting. For this limited version of reporting, reports will be per-event. Future work includes replacing these event-level reports with aggregated reporting.
+
+Edge is only supporting ad auctions that are hosted by the seller’s front end (SFE), within a Trusted Execution Environment (TEE). When auctions are completed, two types of reports are generated via User Defined Functions (UDF):
+
+-   Seller Reports on Auction Results – generated by the function reportResults()
+-   Buyer Report on Auction Winner – generated by the function  reportWin()
+    
+
+The output of the UDFs is a report in the format of [a reporting URL.](https://github.com/privacysandbox/bidding-auction-servers/blob/4a7accd09a7dabf891b5953e5cdbb35d038c83c6/api/bidding_auction_servers.proto#L96C26-L96C26) For example:
+
+    'https://buyer-reporting-server.example/reporting?bid=${bidInfo}
+
+Additionally, reporting is available for interactions within embedded frames. These reports are generated by the reportEvent() API.
+# Buyer and Seller Reports
+## Data from the Browser
+The seller’s ad service receives the following data from the browser:
+
+-   A [Request Blob](https://github.com/WICG/turtledove/blob/36bf1d00641d5f42850ad6f97a64324b53b47c70/FLEDGE_browser_bidding_and_auction_API.md#request-blob-format), contains the following information:
```

Comment:
Nit: prefer not Title Casing things that aren't a proper name. "Request Blob" -> "request blob"